lib/configproxy.js

```diff
@@ -174,6 +175,7 @@ ConfigurableProxy.prototype.add_route = function (path, data) {
     this.routes[path] = data;
     this.trie.add(path, data);
     this.update_last_activity(path);
+    this.statsd.increment('route.add', 1);
```

Comment:
I've moved the noop to inside the constructor. I think the nodejs JIT will optimize away the empty calls.